carrot.html

```diff
@@ -18,11 +18,76 @@
     <div>
       <!-- 시현 -->
     </div>
-    <div>
-      <!-- 지우 -->
+    <div class="woo-part2-1-background">
+      <div class="woo-content-container">
+        <!-- 지우 -->
+        <div class="woo-title-neiborhood">
+          <div class="woo-title-big">이웃과 함께 하는<br />동네생활</div>
+          <div class="woo-title-small">
+            우리 동네의 다양한 이야기를 이웃과 함께 나누어요.
+          </div>
+          <div class="woo-title-icon">
+            <div class="woo-title-buttons">
+              <img
+                class="woo-title-icon-img"
+                src="../Carrot_Market_Cloncoding/images/part2-1_icon1.svg"
```

Comment:
이 경로는 현재 디렉토리에서 한 폴더 더 뒤로 갔다가, 다시 현재 폴더로 들어와서 현재 폴더 안에 있는 images 폴더의 part2-1_icon1.svg 이미지를 의미합니다.

```suggestion
                src="./images/part2-1_icon1.svg"
```

굳이 뒤돌아갔다가 다시 현재 폴더로 들어와서 접근할 필요 없이 바로 같은 위치에 있는 images 폴더에 접근해도 되겠죠!

그리고 현재 git clone을 받고 나서 이 저장소의 이름이 Carrot_Market_Cloncoding -> Carrot_Market_Clonecoding으로 바뀌었기 때문에 이대로 둘 경우 다른 사람들에게는 이미지가 제대로 보이지 않을 수 있어요! 이미지 경로를 수정해봅시다.